### PR TITLE
Giulio/edit prompt text pep8

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd

### CODE REVIEW

File: qualtricssurvey/qualtricssurvey.py [hunk 1]
```diff
@@ -20,20 +22,23 @@ class QualtricsSurvey(StudioEditableXBlockMixin, XBlock):
         display_name=_('Display Name:'),
         default=_('Qualtrics Survey'),
         scope=Scope.settings,
-        help=_('This name appears in the horizontal navigation at the top of the page.'),
+        help=_('This name appears in the horizontal navigation at the top' +
```
Comment (stvstnfrd) [inline qualtricssurvey/qualtricssurvey.py]: Don't do concatenation like this. Instead:

``` python
help=(
    'This name appears in  the horizontal navigation '
    'at the top of the page.'
),
```

File: qualtricssurvey/qualtricssurvey.py [hunk 2]
```diff
@@ -20,20 +22,23 @@ class QualtricsSurvey(StudioEditableXBlockMixin, XBlock):
...
+               'of the page.'),
     )
     survey_id = String(
         display_name=_('Survey ID:'),
         default=_('Enter your survey ID here.'),
         scope=Scope.settings,
-        help=_('This is the ID that Qualtrics uses for the survey, usually an 8 digit number.'),
+        help=_('This is the ID that Qualtrics uses for the survey, which can' +
```
Comment (stvstnfrd) [inline qualtricssurvey/qualtricssurvey.py]: See above note on concatenation.

File: qualtricssurvey/qualtricssurvey.py (hunk 1 above)
Comment (caesar2164) [inline qualtricssurvey/qualtricssurvey.py]: k, what about in the tests file?

File: qualtricssurvey/qualtricssurvey.py [hunk 3]
```diff
@@ -44,9 +49,16 @@ class QualtricsSurvey(StudioEditableXBlockMixin, XBlock):
         display_name=_('Param Name:'),
         default=_('a'),
         scope=Scope.settings,
-        help=_('This is the name for the User ID parameter in the url. If blank, User ID is ommitted from the url.'),
+        help=_('This is the name for the User ID parameter in the url.' +
```
Comment (stvstnfrd) [inline qualtricssurvey/qualtricssurvey.py]: See above note on concatenation.

File: qualtricssurvey/qualtricssurvey.py [hunk 4]
```diff
@@ -44,9 +49,16 @@ class QualtricsSurvey(StudioEditableXBlockMixin, XBlock):
...
+               ' If blank, User ID is ommitted from the url.'),
+    )
+    editable_fields = (
+        'display_name',
+        'survey_id',
+        'your_university',
+        'link_text',
+        'param_name'
```
Comment (stvstnfrd) [inline qualtricssurvey/qualtricssurvey.py]: comma

File: qualtricssurvey/qualtricssurvey.py [hunk 5]
```diff
@@ -103,11 +115,11 @@ def get_resource_url(self, path):
...
-        html_source=None,
-        path_css=None,
-        path_js=None,
-        fragment_js=None,
-    ):
+                       html_source=None,
+                       path_css=None,
+                       path_js=None,
+                       fragment_js=None,
+                       ):
```
Comment (stvstnfrd) [inline qualtricssurvey/qualtricssurvey.py]: I think you're over-indenting here again..

Comment (stvstnfrd) [inline qualtricssurvey/qualtricssurvey.py]: Ya, drop `self` down a line and then unindent the block by 1. The closing comma should align w/ the `d` in `def`.

Comment (caesar2164) [inline qualtricssurvey/qualtricssurvey.py]: i know it looks ridiculous, but pep8 made me, I swear! :smile:

File: qualtricssurvey/tests.py [hunk 6]
```diff
@@ -22,24 +22,28 @@ def make_an_xblock(self, **kw):
...
+        anonymous_user_id.
         """
 
-        xblock = self.make_an_xblock()    
+        xblock = self.make_an_xblock()
         fragment = xblock.student_view()
 
-        url_frag = 'href="https://stanford.qualtrics.com/SE/?SID=Enter your survey ID here.&amp;a='
+        url_frag = 'href="https://stanford.qualtrics.com/SE/?SID=Enter'
+        + ' your survey ID here.&amp;a='
```
Comment (stvstnfrd) [inline qualtricssurvey/tests.py]: This looks odd. I'd prefer:

``` python
url_frag = (
    'blah '
    'blah'
)
```

File: qualtricssurvey/tests.py [hunk 7]
```diff
@@ -22,24 +22,28 @@ def make_an_xblock(self, **kw):
...
+        Checks the student view without param_name;
+        user id part should be missing.
         """
 
         xblock = self.make_an_xblock(param_name=None)
         fragment = xblock.student_view()
 
-        url = '"https://stanford.qualtrics.com/SE/?SID=Enter your survey ID here." target="_blank">click here'
+        url = '"https://stanford.qualtrics.com/SE/?SID=Enter your survey ID'
+        + ' here." target="_blank">click here'
```
Comment (stvstnfrd) [inline qualtricssurvey/tests.py]: See above.

**Comment by stvstnfrd** [comment]
+1

**Pull request merged**